tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27, py32, py33, py34, py35, pypy, style
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/requests_futures
+commands =
+    python test_requests_futures.py
+deps =
+    -r{toxinidir}/requirements-python-2.7.txt
```

Comment:
Yeah, I was just copying `.travis.yml`, but I'll update it to only install the python 2.7 requirements on python 2.7.